packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -507,11 +569,38 @@ export class PluginLedgerConnectorBesu
         return { transactionReceipt: ledgerReceipt };
       default:
         throw new Error(
-          `${fnTag} Unrecognized ReceiptType: ${req.consistencyStrategy.receiptType}`,
+          `${fnTag} Unrecognized ReceiptType: ${request.consistencyStrategy.receiptType}`,
         );
     }
   }
 
+  public async transactPrivate(options: any): Promise<RunTransactionResponse> {
+    if (!this.web3EEA) {
+      await this.initializeEEAClient();
+    }
+
+    const txHash = await this.web3EEA?.eea.sendRawTransaction(options);
```

Comment:
@travis-payne Once you refactor the if statement above to throw instead, the `?` will no longer be necessary because the compiler will understand that the type has been narrowed down sufficiently.